api/data_refinery_api/views.py

```diff
@@ -165,9 +165,16 @@ class ExperimentDocumentView(DocumentViewSet):
     # Is this exhaustive enough?
     search_fields = (
         'title',
+        'publication_title',
         'description',
         'publication_authors',
         'submitter_institution',
+        'accession_code',
+        'alternate_accession_code',
+        'publication_doi',
+        'pubmed_id',
+        'sample_metadata_fields',
```

Comment:
Are you sure we want to be able to search on `sample_metadata_fields`? I thought this was mostly a cached field so we didn't have to calculate it on the fly for every experiment. I think it might end up matching a lot of extra results.